data/hai/unfettered 1 war preparations.txt

```diff
@@ -0,0 +1,1229 @@
+			label changedmind
+			choice
+				`	"I would like a Centipede."`
+					goto centipede
+				`	"This Water Bug seems efficient. I like that."`
+					goto waterbug
+				`	"Actually, I am only here to get a tribute for the Unfettered. These ships seem too costly for me, sorry."`
+			action
+				set "unfettered: too suspect"
+			``
+				goto suspect
+			label centipede
+			`	"You will see it has no equal as cost efficient to transport troops or passengers around! And for a mere 12 million credits."`
+			choice
+				`	"I'll take it, then."`
+				`	"Actually, I changed my mind."`
+					goto changedmind
+			action
+				give ship "Centipede"
+				fine 12000000
+			``
+				goto bought
+			label waterbug
+			`	"A fine ship sir, the best of all the cargo haulers! I can assure you this ship takes a lot more punishment than any human hauler can! And it only costs you 13 million credits."`
+			choice
+				`	"I'll take it, then."`
+				`	"Actually, I changed my mind."`
+					goto changedmind
+			action
+				give ship "Water Bug"
+				fine 13000000
+			label bought
+			action
+				set "unfettered: bought ship"
+			`	He shows you to the ship you requested and tells you all about its different qualities, but you don't really listen - you only bought this ship so he would let you finish your scans. You did have to spend a fine sum for it though... You could always sell it back to someone else, at the shipyard. Maybe you could use this ship's scanners to get an even better scan of the ships here.`
+			label shipbought
+			`	Now that you're in the newly acquired ship, you pretend to look around until he leaves, seeing you are satisfied with your purchase. Then you get back to scanning the shipyard.`
+				goto finishscan
+			label suspect
+			`	"Oh well, that is indeed a most important thing! Please leave this shipyard, immediately." He says this with slightly more intensity, and you wonder if it's a threat.`
+			choice
+				`	(Pretend to leave, and then come back to finish the scans.)`
+				`	(Leave without the scans.)`
+					decline
+			label finishscan
+			branch toosuspect
+				has "unfettered: too suspect"
+			`	You manage to finish the scans without any more interruption, and are eager to get back to your ship to get all of this over with.`
+				accept
+			label toosuspect
+			action
+				fine 100000
+				clear "unfettered: too suspect"
+			`	It looks like you have aroused too much suspicion. Maybe he was waiting to see if you would actually leave, or maybe he alerted the local authorities, but the result is the same: they arrest you and even fine you <fine>.`
+				decline
+	on decline
+		fail "Unfettered: Solifuge 6: Shipyard Upgrade"
+
+
+
+mission "Unfettered: Solifuge 8: Useful Korath Scientists"
```

Comment:
I'm going to have to go over this mission later, as it deals with direct communication with the Korath.  You've got some of the earlier concepts we talked about down, but then there's the writing, drawing, and gesticulating.

I imagine this translator ignores this and focuses on verbalisations, hence the incompleteness of the translation, that's great.  I think you could lean into it further if you describe some of the somatic and visual components to the player.

I could do that for you, but as it's a rather large thing and it's your story I thought I should afford you the chance to have a go at it first.  Plus it's almost 0300 here and I'm running out of steam :-)

Reply:
Honestly the more this is a collective project the happier I am for it!
I'll give it a shot still but really, feel free to be critical about it.